root/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# based on https://github.com/dask/dask-docker/blob/master/base/Dockerfile
+# but more permissive about image size due to read-only requirement in openshift
+# FROM daskdev/dask:2.9.0
+ARG ROOTVER=6.32.04
+ARG PLATFORM=ubuntu24.04
+FROM rootproject/root:${ROOTVER}-${PLATFORM}
+ARG CERTIFICATE_VERSION=1.134IGTFNEW
```

Comment:
Add a comment on what this is so someone coming in later can read this file and understand it?

Reply:
On what is? The certificate version?

Reply:
Yeah - that just looks like a super magic number. :-)